beacon-chain/rpc/eth/light-client/handlers_test.go

```diff
@@ -2047,3 +1884,121 @@ func TestLightClientHandler_GetLightClientEventBlock_NeedFetchParent(t *testing.
 	require.Equal(t, true, syncAggregate.SyncCommitteeBits.Count() >= minSignaturesRequired)
 	require.Equal(t, slot-1, eventBlock.Block().Slot())
 }
+
+// setupDB instantiates and returns a Store instance.
+func setupDB(t testing.TB) *kv.Store {
+	db, err := kv.NewKVStore(context.Background(), t.TempDir())
+	require.NoError(t, err, "Failed to instantiate DB")
+	t.Cleanup(func() {
+		require.NoError(t, db.Close(), "Failed to close database")
+	})
+	return db
+}
+
+func createUpdate(t *testing.T, v int) (interfaces.LightClientUpdate, error) {
```

Comment:
Is this function the same as the other `createUpdate`? If yes, you can move it from here to `testing/util/lightclient.go`

Reply:
did it for the `handlers_test.go` but it can't be done for `kv/lightclient_test.go` because of circular import